### PR TITLE
[MRG] Update server proxy and rsession proxy

### PR DESCRIPTION
**Pull request by betatim**
Both haven't been updated in a long time (~1yr) so we should refresh
them. This brings both to the latest commit in their repos.

Can anyone remember if there were big changes in either that would break things for users of repo2docker?

### CODE REVIEW

File: repo2docker/buildpacks/_r_base.py [hunk 1]
```diff
@@ -55,8 +55,8 @@ def rstudio_base_scripts():
             "${NB_USER}",
             # Install nbrsessionproxy
             r"""
-                pip install --no-cache-dir https://github.com/jupyterhub/jupyter-server-proxy/archive/7ac0125.zip && \
-                pip install --no-cache-dir jupyter-rsession-proxy==1.0b6 && \
+                pip install --no-cache-dir https://github.com/jupyterhub/jupyter-server-proxy/archive/00be53dfed8ab5a0baaf8332e3fc6bd44dc5bf1a.zip && \
```
Comment (consideRatio) [inline repo2docker/buildpacks/_r_base.py:58]: I've recently made a 1.3.1 release, should we use that?
https://pypi.org/project/jupyter-server-proxy/

**Comment by consideRatio** [comment]
> Can anyone remember if there were big changes in either that would break things for users of repo2docker?

I've not followed the changes for that long, I'm not sure, but I know there is no changelog available. But, thanks to @choldgraf's github-activity project, I present the results below from running:

```
github-activity --kind=pr -o prs.md jupyterhub/jupyter-server-proxy --since 7ac0125
```

# 7ac0125...master@{2020-04-04}
([full changelog](https://github.com/jupyterhub/jupyter-server-proxy/compare/7ac0125...00be53dfed8ab5a0baaf8332e3fc6bd44dc5bf1a))


## Merged PRs
* Update readme with labextension info [#187](https://github.com/jupyterhub/jupyter-server-proxy/pull/187) ([@consideRatio](https://github.com/consideRatio))
* Quote as little as needed for a valid proxied web request [#186](https://github.com/jupyterhub/jupyter-server-proxy/pull/186) ([@consideRatio](https://github.com/consideRatio))
* Capture request timeout error and increase the timeout value [#183](https://github.com/jupyterhub/jupyter-server-proxy/pull/183) ([@oeway](https://github.com/oeway))
* Proxy Websocket subprotocols [#180](https://github.com/jupyterhub/jupyter-server-proxy/pull/180) ([@xhochy](https://github.com/xhochy))
* Quote proxied_path to make it a safe URL (fixes #138) [#177](https://github.com/jupyterhub/jupyter-server-proxy/pull/177) ([@oeway](https://github.com/oeway))
* Support usage of JupyterLab 2.0.0 [#174](https://github.com/jupyterhub/jupyter-server-proxy/pull/174) ([@consideRatio](https://github.com/consideRatio))
* Fix Traitlets link in server process docs [#170](https://github.com/jupyterhub/jupyter-server-proxy/pull/170) ([@jtpio](https://github.com/jtpio))
* Update NPM badge for new npm repo [#166](https://github.com/jupyterhub/jupyter-server-proxy/pull/166) ([@consideRatio](https://github.com/consideRatio))
* Update npm package name for JupyterLab [#165](https://github.com/jupyterhub/jupyter-server-proxy/pull/165) ([@segevfiner](https://github.com/segevfiner))
* Replace icon_url with icon_path in __init__.py. [#163](https://github.com/jupyterhub/jupyter-server-proxy/pull/163) ([@ka-zo](https://github.com/ka-zo))
* Bump version number [#161](https://github.com/jupyterhub/jupyter-server-proxy/pull/161) ([@yuvipanda](https://github.com/yuvipanda))
* Add badges [#160](https://github.com/jupyterhub/jupyter-server-proxy/pull/160) ([@consideRatio](https://github.com/consideRatio))
* Move npm package to jupyterlab org [#159](https://github.com/jupyterhub/jupyter-server-proxy/pull/159) ([@blink1073](https://github.com/blink1073))
* Allow proxying to remote host [#154](https://github.com/jupyterhub/jupyter-server-proxy/pull/154) ([@jacobtomlinson](https://github.com/jacobtomlinson))
* Optionally map requested → proxied paths [#151](https://github.com/jupyterhub/jupyter-server-proxy/pull/151) ([@manics](https://github.com/manics))
* Add code-server-traitlet example. [#148](https://github.com/jupyterhub/jupyter-server-proxy/pull/148) ([@ryanlovett](https://github.com/ryanlovett))
* Fix WebSocket race condition [#140](https://github.com/jupyterhub/jupyter-server-proxy/pull/140) ([@tibdex](https://github.com/tibdex))
* Support websocket compression [#136](https://github.com/jupyterhub/jupyter-server-proxy/pull/136) ([@rcolle](https://github.com/rcolle))
* Fix typo base_url on arbitrary-ports [#135](https://github.com/jupyterhub/jupyter-server-proxy/pull/135) ([@heatwole](https://github.com/heatwole))
* Update for JL 1.0-alpha [#134](https://github.com/jupyterhub/jupyter-server-proxy/pull/134) ([@athornton](https://github.com/athornton))
* install-doc-update [#132](https://github.com/jupyterhub/jupyter-server-proxy/pull/132) ([@quasiben](https://github.com/quasiben))
* Allow for other hosts than `localhost` to be proxied. [#128](https://github.com/jupyterhub/jupyter-server-proxy/pull/128) ([@ian-r-rose](https://github.com/ian-r-rose))
* WebsocketHandler.get is async [#119](https://github.com/jupyterhub/jupyter-server-proxy/pull/119) ([@rmorshea](https://github.com/rmorshea))

File: repo2docker/buildpacks/_r_base.py (hunk 1 above)
Comment (manics) [reply repo2docker/buildpacks/_r_base.py:58]: :+1: If there's a new improvement in that justifies using a commit I think that's a good reason to make a new release of `jupyter-server-proxy` so everyone can use it, not just repo2docker.

Comment (consideRatio) [inline repo2docker/buildpacks/_r_base.py:58]:
```suggestion
                pip install --no-cache-dir jupyter-server-proxy==1.3.1 && \
```

**Comment by betatim** [comment]
@consideRatio can you make a new release and add 1.3.2 to https://github.com/jupyterhub/jupyter-server-proxy/releases? The reason I updated server proxy was to see if the increased timeout deals with https://github.com/jupyterhub/jupyter-server-proxy/issues/146. It seems it does. At least I don't get an error locally any more. The timeout change hasn't been released yet though. (I am perfectly fine using a particular commit instead of a release though, so only make a new release if you have time.)

File: repo2docker/buildpacks/_r_base.py (hunk 1 above)
Comment (betatim) [reply repo2docker/buildpacks/_r_base.py:58]: I applied this but then removed it again because 1.3.1 doesn't include the increased timeout.

**Comment by consideRatio** [comment]
@betatim ok!

@ryanlovett @yuvipanda I'm making a new release of jupyter-server-proxy containing a single PR that I consider a bugfix, so from 1.3.1 to 1.3.2. Guided by the principle in release.rst saying:

> We should release early & often, following semver wherever possible.

File: repo2docker/buildpacks/_r_base.py (hunk 1 above)
Comment (consideRatio) [reply repo2docker/buildpacks/_r_base.py:58]:
```suggestion
                pip install --no-cache-dir jupyter-server-proxy==1.3.2 && \
```

**Pull request merged**